relayer/flask/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 class FlaskRelayer(object):
 
-    def __init__(self, app: Flask, logging_topic: str, kafka_hosts: str = None, **kwargs: str) -> None:
+    def __init__(self, app: Flask = None, logging_topic: str = None, kafka_hosts: str = None, **kwargs: str) -> None:
```

Comment:
isn't `kwargs` a dict?

Reply:
yup, lemme fix that also

Reply:
👍

Reply:
For some reason changing it to `Dict` causes it to fail 😢 
I also had to change the init_app argument to `Any` for it to pass the test.

Reply:
did you change both kwargs to dict here and in `init_app`?

Reply:
and everywhere... there is a lot of places where kwargs is typed as a `str` :s

Reply:
let me try a full refactor. If it takes more than 30 min then its probably not worth it. :loading:

Reply:
nope it will take forever 🤒 😭

Reply:
Opened [this issue](https://github.com/wizeline/relayer/issues/46) to dedicate more time to debugging this later on.

Reply:
Remember that `*` and `**` is for unpacking sequences and dicts respectively. So... the typing is not a `Dict` but a description of the contained objects. So, when you type as `str` it is expecting that all elements are `str` (http://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html)
```py
# This makes each positional arg and each keyword arg a "str"
def call(self, *args: str, **kwargs: str) -> str:
    request = make_request(*args, **kwargs)
    return self.do_api_query(request)
```

So, the approaches to proceed is:
1. Be explicit about which values are you receiving
2. Use `override` decorator
3. Use `Any` for describing the content